ACKNOWLEDGEMENTS.md

```diff
@@ -39,6 +38,9 @@ the [Department of Computer Science and Engineering] at the
 Thanks very much to [Intel] for donating compiler licenses during the
 initial years of the project for the purpose of portability testing.
 
+Thanks to Google for funding several students to work on projects within
+Chapel as part of the [Google Summer of Code] program.
+
```

Comment:
This looks good, but I tend to try and keep these listed approximately reverse chronologically (with some focus also on significance).  So how about moving this just under the GitHub/SourceForge paragraph?